services/payments/src/simcore_service_payments/api/rest/_exceptions.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from fastapi import HTTPException, Request, status
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import JSONResponse
+
+from ...models.schemas.errors import DefaultApiError
+
+_logger = logging.getLogger(__name__)
+
+
+# NOTE:  https://www.starlette.io/exceptions/
+# Handled exceptions **do not represent error cases** !
+# - They are coerced into appropriate HTTP responses, which are then sent through the standard middleware stack.
+# - By default the HTTPException class is used to manage any handled exceptions.
+
+
+async def http_exception_as_json_response(
+    request: Request, exc: HTTPException
+) -> JSONResponse:
+    assert request  # nosec
+    error = DefaultApiError.from_status_code(exc.status_code)
+
+    error_detail = error.detail or ""
```

Comment:
thought: we should have a module in the servicelib with these at some point no? I have the feeling this looks like a deja-vu

Reply:
Agreed. My plan is to move some of the utilities later to `servicelib.fastapi`